platform/linux-generic/include/odp_pktio_ops_subsystem.h

```diff
@@ -88,10 +89,40 @@ typedef ODP_MODULE_CLASS(pktio_ops) {
 /* Maximum size of pktio specific ops data.*/
 #define ODP_PKTIO_ODPS_DATA_MAX_SIZE 80000
 
+/* Pktio ops data pool */
+#define ODP_PKTIO_OPS_DATA_POOL_SIZE 160000
+#define ODP_PKTIO_OPS_DATA_POOL_NAME "ODP_PKTIO_OPS_DATA"
+
 /* Extract pktio ops data from pktio entry structure */
 #define odp_ops_data(_p, _mod) \
 	((pktio_ops_ ## _mod ## _data_t *)(uintptr_t)_p->s.ops_data)
 
+#define odp_ops_data_alloc(_p, _mod)					\
+({ \
+	odpdrv_shm_pool_t _pool = ODPDRV_SHM_POOL_INVALID;		\
+	uint64_t _size = sizeof(pktio_ops_ ## _mod ## _data_t);		\
```

Comment:
Temporarily, roundup the size to cache line size. This way all the memory allocations will be cache line aligned.